Framework/Algorithms/test/GetEiTest.h

```diff
@@ -272,4 +280,57 @@ class GetEiTest : public CxxTest::TestSuite {
   }
 };
 
+class GetEiTestPerformance : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static GetEiTestPerformance *createSuite() {
+    return new GetEiTestPerformance();
+  }
+  static void destroySuite(GetEiTestPerformance *suite) { delete suite; }
+
+  void setUp() {
+    getEiTest = GetEiTest();
+    inputWS1 = getEiTest.createTestWorkspaceWithMonitors();
+    outputName1 = "eitest1";
+    AnalysisDataService::Instance().add(outputName1, inputWS1);
+
+    inputWS2 = getEiTest.createTestWorkspaceWithMonitors();
```

Comment:
Is it okay to do that, instead of re-writing the methods or separating them in a different class?